### ISSUE TITLE
IE Driver Beta: initialBrowserUrl capability is ignored, and its default value has changed

### ISSUE THREAD

**Issue by juangj** [SeleniumHQ/selenium#3715]
## Meta -
OS:  Windows 7
Selenium Version: a little after 3.3 (commit e9a906a347dcdb9991c7705fa4a291152982c5be), though it doesn't much matter
Browser:  Internet Exporer
Browser Version:  9.0.8112.16421

IE Driver Beta Version: 3.3.99.1

## Expected Behavior -
The `initialBrowserUrl` capability should be set to the value specified in the New Session request.

Also, the default value when it is not specified used to be `"http://localhost:5555/"`, which is the "This is the initial start page for the WebDriver server" page.
 
## Actual Behavior -
The default value of `initialBrowserUrl` (inside the `se:ieOptions` object) is now empty (`""`), and the browser starts on whatever the default homepage is.

Even if I specify an `initialBrowserUrl` capability in the `alwaysMatch` section, it is disregarded.

I do understand this capability is something of a "best-effort" capability, and I can work around this if need be.

## Steps to reproduce -
I'm using the Selenium Standalone Server and just curling commands to the driver to send a minimal set of capabilities:

`curl -sd'{"desiredCapabilities": {"browserName": "internet explorer", "se:ieOptions": {"initialBrowserUrl": "http://www.example.com/"}}}' http://localhost:26657/wd/hub/session`

The Selenium Standalone Server translates that request into:

```json
{
  "desiredCapabilities": {
    "se:ieOptions": {
      "initialBrowserUrl": "http://www.example.com/"
    },
    "browserName": "internet explorer"
  },
  "requiredCapabilities": {},
  "capabilities": {
    "desiredCapabilities": {
      "se:ieOptions": {
        "initialBrowserUrl": "http://www.example.com/"
      },
      "browserName": "internet explorer"
    },
    "requiredCapabilities": {}
  },
  "alwaysMatch": {
    "se:ieOptions": {
      "initialBrowserUrl": "http://www.example.com/"
    }
  },
  "firstMatch": [
    {
      "browserName": "internet explorer"
    }
  ]
}
```

but the response is (with irrelevant keys omitted):

```json
{
    "value": {
        "capabilities": {
            "se:ieOptions": {
                "initialBrowserUrl": ""
            }
        },
        "sessionId": "a5d84976-e8f1-4bae-b887-9dc2dc3aeca0"
    }
}
```

**Comment by juangj** [comment]
Full request/response in [iedriver.log.txt](https://github.com/SeleniumHQ/selenium/files/869617/iedriver.log.txt)

**Comment by juangj** [comment]
For what it's worth, I can't get other se:ieOptions capabilities to take either. For example:
```json
"se:ieOptions": {
    "ie.ensureCleanSession": true
}
```

doesn't seem to have an effect. The returned driver says `ie.ensureCleanSession: false`.

**Comment by jimevans** [comment]
Oh, hang on. That's not a spec-compliant payload for the new session command. The `alwaysMatch` and `firstMatch` properties should be part of the object assigned to the `capabilities` property, not top-level properties of the entire payload. This is a Java language bindings bug.

**Comment by jimevans** [comment]
Fixed in f8b23bb
